CHANGES/563.bugfix

```diff
@@ -0,0 +1 @@
+Fix issue with semicolon as a separator in query parameters.
```

Comment:
As it stands now, this PR also adds a feature of passing a separator arg to the constructor. So this probably shouldn't be a `.bugfix` file.

Reply:
Also, this note is supposed to be user-oriented. When somebody reads the changelog, it should give them some idea of what this is about. It currently does not name the issue, nor does it mention the way it's addressed and what the impact is for the users.